.github/workflows/camunda-formio-tasklist-vue-cd.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Camunda Formio Task List
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - master
+    paths:
+      - "camunda-formio-tasklist-vue/**"
+defaults:
+  run:
+    shell: bash
+    working-directory: ./camunda-formio-tasklist-vue
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm ci
+      # - run: npm test
+      - run: npm install
```

Comment:
just FYI: npm ci will create modules out of the package.lock file which creates the same node modules for every run, npm install create node modules out of package.json which might update the package.lock file on every run, so if we are following a habit of committing package.lock better to use npm ci only for the pipeline. @abhilash-aot @shuhaib-aot 
what are your thoughts?

Reply:
yes we may not need both npm ci and npm i . Only one is enough @sumesh-aot